### ISSUE TITLE
Proofread MUFI tables

### ISSUE THREAD

**Issue by scottkleinman** [WheatonCS/Lexos#458]
Building on issue #355...

The code points in the MUFI tables, particularly MUFI 4.0, need proofreading. I think the main problem is that in some cases Excel has removed leading zeros or changed it to its patented scientific formatting&trade;.

**Comment by scottkleinman** [comment]
Update: I have proofread MUFI 3.0, though I still have to do MUFI 4.0 before I can close this issue. However, I have improved the UI for viewing rule sets considerably. Rule sets are now stored in JSON files and loaded into Datatables by Ajax. This allows for sorting and searching. One consequence, however, is that all the tables have eight columns since that's what the MUFI tables contain. For the other two rule sets, the columns are just left blank. We could fill in some of the information that would go there at a later date.

**Comment by scottkleinman** [comment]
Hi have now checked the MUFI 4.0 table. There were quite a few code points that lost their zero padding because of Excel. In replacing it, I inadvertently turned Unicode glyphs into Python escapes (e.g. `\uf1ea` for the _punctus versus_ symbol &#xF1EA;, which won't display it GitHub's font). Both Chrome and Firefox seem to display it just as well, so I left it. The MUFI 3.0 table still has the glyphs.

Regardless, I think both tables are now accurate enough to close this issue. At some point, we may want to run a quick algorithmic text to ensure that Lexos' internal tables correspond to all the code points in the JSON source of the display tables.
